src/foam/u2/view/ValueView.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'ValueView',
+  extends: 'foam.u2.View',
+
+  documentation: 'Just shows the value of data as a string.',
+
+  methods: [
+    {
```

Comment:
Just:
methods: [
  function initE() {
        return this.add(this.data);
   }
]

would be shorter

Reply:
Resolved by a1584561a